backend/login/urls.py

```diff
@@ -3,5 +3,6 @@
 
 urlpatterns = [
     url(r'^$', login_page),
-    url(r'^mysnu/', LoginRequest.as_view()),
+    url(r'^(?P<option>mysnu)/', LoginRequest.as_view()),
+    url(r'^(?P<option>file)/', LoginRequest.as_view()),
```

Comment:
This would better be `url(r'^(?P<option>mysnu|file)/', LoginRequest.as_view()),`